### PR TITLE
Implement two fractions #154

### LINKED ISSUE #154: Implement two fractions and assign players to them

### PR DESCRIPTION
**Pull request by przenz**
Links to #154

### CODE REVIEW

File: client/gameObjects/Skeleton.js [hunk 1]
```diff
@@ -46,10 +46,12 @@ const MIN_TICK = 8;
 const MAX_TICK = 12;
 
 const HEALTH_BAR_OFFSET_X = -32;
-const HEALTH_BAR_OFFSET_Y = -56;
+const HEALTH_BAR_OFFSET_Y = -36;
```
Comment (neu5) [inline client/gameObjects/Skeleton.js:49]: I'm not sure about this position change.
Player will see only allies health bar (#142) so for the others (enemies) will see only the name.
Without health bars it looks like this
image

The space between names and the players is quite large

File: client/gameObjects/Skeleton.js [hunk 2]
```diff
@@ -154,14 +158,19 @@ export default class Skeleton extends Phaser.GameObjects.Image {
     this.frame = this.texture.get(this.direction.offset);
 
     this.label = this.scene.add
-      .text(this.x, this.y, this.displayName)
+      .text(
+        this.x,
+        this.y,
+        `[${this.fraction.charAt(0)}] ${this.displayName}`,
```
Comment (neu5) [inline client/gameObjects/Skeleton.js]: make it a method getName or something like that

File: public/assets/map/gameObjects.js [hunk 3]
```diff
@@ -9,16 +9,16 @@ const getStartingTileTree = getStartingTile;
 
 module.exports = [
   {
-    name: "house1",
-    displayName: "house",
+    name: "house_dire",
```
Comment (neu5) [inline public/assets/map/gameObjects.js]: Please make something like
`const FRACTIONS = [
  {
    name: "Radiant",
    houses: [{ name: "house_radiant", displayName: "Radiant Sanctuary"}]
  },
];`
and use it here and in `server/mocks/players.js`

File: server/mocks/players.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ module.exports = [
   {
     name: "player1",
     displayName: "player 1",
+    fraction: "Radiant",
```
Comment (neu5) [inline server/mocks/players.js]: take it from constant

File: server/mocks/players.js [hunk 5]
```diff
@@ -20,7 +21,7 @@ module.exports = [
       follow: true,
       fight: true,
       showRange: false,
-      respawnBuilding: "house1",
+      respawnBuilding: "house_radiant",
```
Comment (neu5) [inline server/mocks/players.js]: take it from constant

**Review by neu5 — CHANGES_REQUESTED** [review]

File: client/gameObjects/Skeleton.js (hunk 1 above)
Comment (przenz) [reply client/gameObjects/Skeleton.js:49]: This should be resolved in the ticket addressing health bar visibility.

File: client/gameObjects/Skeleton.js [hunk 6]
```diff
@@ -178,6 +173,10 @@ export default class Skeleton extends Phaser.GameObjects.Image {
     height: 80,
   };
 
+  getName() {
```
Comment (neu5) [inline client/gameObjects/Skeleton.js]: getDisplayName ?

File: public/assets/map/gameObjects.js [hunk 7]
```diff
@@ -7,18 +9,21 @@ const getStartingTileHouse = ({ tileX, tileY }) => ({
...
     type: "House",
     positionTile: { tileX: 14, tileY: 8 },
     startingTile: getStartingTileHouse,
     size: { tileX: 4, tileY: 3 },
   },
   {
-    name: "house_radiant",
-    displayName: "Radiant Sanctuary",
+    name: radiantHouse.name,
+    displayName: radiantHouse.name,
```
Comment (neu5) [inline public/assets/map/gameObjects.js]: name => displayName

**Review by neu5 — CHANGES_REQUESTED** [review]

**Review by neu5 — APPROVED** [review]

**Pull request merged**